basket/base/tests/test__utils.py

```diff
@@ -4,11 +4,7 @@
 from django.test.utils import override_settings
 
 from basket.base.utils import email_is_testing
-from basket.settings import (
-    SENSITIVE_FIELDS_TO_MASK_ENTIRELY,
-    SENSITIVE_FIELDS_TO_MASK_PARTIALLY,
-    before_send,
-)
+from basket.settings import SentryConfigurationMixin
```

Comment:
Should app (or test) code be using a mixin directly?

Reply:
Do you see an issue with it? It's only named "Mixin" to hint that it can be added as a base class for the other classes. The tests were importing from settings before for testing the sentry config, and it's doing the same after this changeset.

Reply:
Only issue I see is that it's not testing what the code will be using (which I assume remains `settings`) and just looks in what's in a class, which could or could not be included in the actual settings. Would it be better to include this mixin with the testing config? Honestly, looking at these tests they seem mostly superfluous.